mesheryctl/internal/cli/root/error.go

```diff
@@ -0,0 +1,45 @@
+package root
+
+import "github.com/layer5io/meshkit/errors"
+
+const (
+	ErrProcessingConfigCode        = "1050"
+	ErrCreatingConfigFileCode      = "1051"
+	ErrAddingTokenToConfigCode     = "1052"
+	ErrAddingContextToConfigCode   = "1053"
+	ErrUnmarshallingConfigFileCode = "1054"
+	ErrGettingRequestContextCode   = "1055"
+	ErrInvalidAPIResponseCode      = "1056"
+	ErrUnmarshallingAPIDataCode    = "1057"
+	ErrConnectingToServerCode      = "1058"
+)
+
+var (
+	ErrCreatingConfigFile = errors.New(ErrCreatingConfigFileCode, errors.Alert, []string{"Unable to create config file"}, []string{"Unable to create config file"}, []string{}, []string{})
```

Comment:
@hexxdump, we could use some help on these - filling in the probable cause and suggested remediation.

Reply:
@gr455 will you open an issue to track the need for this information for these mesheryctl error codes? And, we'll move forward with merge here.

Reply:
Thank you. 👍